### PR TITLE
Add notification test mode

### PR DESCRIPTION
**Pull request by kobakei**
## Issue
-

## Overview (Required)
- Add notification test mode using debot
  - If notification test mode is enabled, notifications will be displayed 5 sec after FAB check actions.

## Links
-

## Screenshot
Before | After
:--: | :--:
 | device-2017-02-09-175610

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java [hunk 1]
```diff
@@ -50,6 +54,7 @@ public void onCreate() {
 
         DebotStrategyBuilder builder = new DebotStrategyBuilder.Builder(this)
                 .registerMenu("Clear cache", clearCache)
+                .registerMenu("Test notification", notificationStrategy)
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java]: I think it's better to change title by test mode on/off.

File: app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java [hunk 2]
```diff
@@ -50,6 +54,7 @@ public void onCreate() {
 
         DebotStrategyBuilder builder = new DebotStrategyBuilder.Builder(this)
                 .registerMenu("Clear cache", clearCache)
+                .registerMenu("Notification test ON/OFF", notificationStrategy)
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java]: Sorry, I mean like this.

- If notification test mode is ON, the title should be "Start notification test mode".
- If notification test mode is OFF, the title should be "Stop notification test mode".

Is it possible to check pref here?

Comment (kobakei) [inline app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java]: OK, I will fix.

Comment (kobakei) [inline app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java]: I'm not sure whether the debug menu tile can be changed at runtime.
It depends on Debot spec.

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java]: Ah, I see.

File: app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java [hunk 3]
```diff
@@ -26,6 +27,9 @@
     @Inject
     ClearCache clearCache;
 
+    @Inject
+    NotificationStrategy notificationStrategy;
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java]: I think it's not good to increase Strategy classes to Application class.
I'll create Provider class for debot for the future.

File: app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java (hunk 2 above)
Comment (kobakei) [inline app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java]: I have solved this issue by re-initializing debot when test menu is pressed.

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java]: Thanks!

**Comment by konifar** [comment]
LGTM! Thanks 👍

**Pull request merged**